WeakAurasOptions/SubRegionOptions/Glow.lua

```diff
@@ -0,0 +1,263 @@
+if not WeakAuras.IsCorrectVersion() then return end
+
+local SharedMedia = LibStub("LibSharedMedia-3.0");
+local L = WeakAuras.L;
+
+local screenWidth, screenHeight = math.ceil(GetScreenWidth() / 20) * 20, math.ceil(GetScreenHeight() / 20) * 20;
+
+
+local hiddenGlowExtra = function()
```

Comment:
shouldn't it be moved into "createOptions" and use as 3rd parameter `"glowextra" .. index` and add `.. index` on every SetCollapse or IsCollapsed ?